boot/seal.go

```diff
@@ -53,18 +53,17 @@ var (
 	HasFDESetupHook = func() (bool, error) {
 		return false, nil
 	}
-	RunFDESetupHook = func(op string, params *FdeSetupHookParams) error {
-		return fmt.Errorf("internal error: RunFDESetupHook not set yet")
+	RunFDESetupHook = func(op string, params *FdeSetupHookParams) ([]byte, error) {
```

Comment:
we probably want to add to the comment(s) about these hooks that they are run with the state lock held

Reply:
btw that struct needs also to be renamed s/Fde/FDE/